frontend/src/container/IngestionSettings/MultiIngestionSettings.tsx

```diff
@@ -70,7 +71,10 @@ const { Option } = Select;
 
 const BYTES = 1073741824;
 
-export const disabledDate = (current: Dayjs): boolean =>
+// Using any type here because antd's DatePicker expects its own internal Dayjs type
+// which conflicts with our project's Dayjs type that has additional plugins (tz, utc etc).
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+export const disabledDate = (current: any): boolean =>
```

Comment:
If you have a list of types that we need then plus antds' then we can make a interface and maybe pass here, as Partial also if required.

Reply:
@SagarRajput-7, @YounixM, The issue here is that we have configured additional plugins for Dayjs(e.g. tz, utc) that are necessary for timezone. but the ant design's Dayjs type doesn't recognize the additional properties that are introduced by these plugins. 

I tried 

```typescript
type TimezoneAgnosticDate = Omit<
	Dayjs,
	'tz' | 'offsetName' | 'utc' | 'local' | 'isUTC'
>;
```

but it didn't work. 

The last resort was to use `any`.

A similar issue has been reported here as well: https://github.com/iamkun/dayjs/issues/942